README.md

```diff
@@ -89,6 +89,21 @@ Once you have either downloaded the VSIX file or created it you can now install
 
 Please refer to the Wiki page at https://github.com/apache/daffodil-vscode/wiki
 
+To debug a daffodil schema, choose open a folder from the File menus, then select a dfdl.xsd file. Once the file loads in VS Code, then set a desired breakpoint.
```

Comment:
We have a link to the wiki which looks to show very similar things. I'm not sure we need them both in the wiki and here. The wiki seems to have more information.

Reply:
I'm not sure what you are saying, are you saying the wiki is all we need?  Or are you saying to move the wiki info to the readme? Or something else?

Reply:
I don't really have an opinion if the information lives in the wiki or lives in the repo. What I'm saying is we should not have the same information in both the wiki *and* the repo. That's too much of a maintenance burden. The information on the wiki seems more detailed, so I would suggest we just keep it there.